src/store/dashboard/awsCloudDashboard/awsCloudDashboardWidgets.ts

```diff
@@ -60,13 +60,13 @@ export const computeWidget: AwsCloudDashboardWidget = {
 
 export const costSummaryWidget: AwsCloudDashboardWidget = {
   id: getId(),
-  titleKey: 'aws_cloud_dashboard.cost_title',
+  titleKey: messages.AWSComputeTitle,
```

Comment:
Should use the cost title, not the compute title

The old key was "Amazon Web Services filtered by OpenShift cost"

Reply:
Done